test/cases/000_config/010_kmod/kmod.yml

```diff
@@ -13,5 +13,13 @@ onboot:
      - /lib/modules:/lib/modules
     capabilities:
      - all
+services:
```

Comment:
unfortunately that doesn't work... if the poweroff container runs in onboot, the machine doesn't get powered off. haven't debugged why just yet or whether this is intentional or not but @justincormack would know for sure.